PR script for V

JohnDoeefkfmoe opened the pull request.

Commit:
first file to top GH

Commit:
add file afterssh.txt

Commit:
add stress script

Commit:
add vptest.txt to VPR

JohnDoeefkfmoe committed:
Merge pull request #1 from JohnDoeefkfmoe/VPR

add vptest.txt to VPR

Commit:
add #test

Commit:
final

tritonJS826 commented on stress_script_beforeGH.sh:
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+TYPE=$1  # Что нагружем: cpu или ram
+VALUE=$2 # Количесство потоков или памяти в MB
```
для того, чтобы удобнее в будущем было пользоваться скриптом удобно сделать вспомогательный хелп, чтобы скрипт был интерактивным (мол введите тип нагрузки: 1 - cpu, 2 - ram etc.) рекомендую) это сделает возможным использование этого скрипта другими пользователями!

tritonJS826 replied:
$1 $2 возможно не самыйлучший подход для доступа - используй флаги (например чтобы вызывать так: `./stress_script_beforeGH.sh --type cpu, ram --ram-value 5000 --threads 8`)

tritonJS826 commented on stress_script_beforeGH.sh:
```diff
@@ -0,0 +1,51 @@
+# создание нагрузки на cpu
+stress_cpu() {
+    echo "Нагрузка CPU на $VALUE потоков"
+    for i in $(seq 1 "$VALUE"); do
+        while :; do :; done &
+    done
```
у тебя 2 цикла вложенных. Попробуй отрефачить- нагрузку на 1 поток вынести в отдельную вункцию - тогда ее можно будет переиспользовать в других сценариях!

tritonJS826 commented on stress_script_beforeGH.sh:
```diff
@@ -0,0 +1,51 @@
+    exit 1
+fi
+# создание нагрузки на cpu
+stress_cpu() {
```
в идеале останавливать команду не через ctrl+c, а например по таймауту, тогда такие нагрузки можно будет автоматизировать со временем

tritonJS826 reviewed (commented):
В целом достаточно красиво все!

Commit:
+ теперь сбор аргументов происходит в работе с флагами
+ добавил timeout
+ добавил значения по умолчанию, если не вводятся значения от пользователя
+ добавил help
+ местами скрипт переделан, т.к. при внесении изменений та или иная часть переставала работать
